### DISCUSSION TITLE
Different color for latent workers on builders view

### DISCUSSION THREAD

**Discussion by mscastanho** [buildbot/buildbot#6532] [Ideas]
Knowing if there are any offline workers by glancing over the last column of the builders page in the web UI is very useful. I can quickly check if there are any red bubbles, which means I need to take action to restart those workers.

But latent workers (like DockerLatentWorker) also appear in red, just like any other offline workers. It makes sense to be this way, after all the worker is not deployed, so it's indeed offline. But I wonder if it would be possible to use a different color for inactive latent workers (like blue/orange?). 

This way it would be much easier to distinguish between regular offline workers (that may need to be restarted) and latent workers (ok to be offline and so nothing to do).

**Comment by vibbo** [comment]
Hi. It would also be nice to have different color for latent workers that are in process on insubstantiation.
